src/tests/homework.spec.js

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from "@playwright/test";
+
+import {
+  ApplicationTexts,
+  invalidPassword,
+  password,
+  userFullName,
+} from "../fixtures/fixtures.js";
+
+import { LoginPage } from "./pages/login.page.js";
+import { RegistrationPage } from "./pages/registration.page.js";
+
+test.describe("Registration Page is Correctly Displayed", async () => {
+  test.beforeEach(async ({ page }) => {
+    const loginPage = new LoginPage(page);
+    await loginPage.open();
+    await loginPage.registrationButton.click();
+    await test
+      .expect(page)
+      .toHaveTitle(ApplicationTexts.registrationPage.title);
+  });
+
+  test("verify registration form is displayed correctly ", async ({ page }) => {
+    const registrationPage = new RegistrationPage(page);
+
+    await expect(
+      registrationPage.fullNameInput,
+      "name input field should be visible"
+    ).toBeVisible();
+    await expect(
+      registrationPage.fullNameInput,
+      "name input field should be enabled"
+    ).toBeEnabled();
+
+    await expect(
+      registrationPage.emailInput,
+      "email input field should be visible"
+    ).toBeVisible();
+    await expect(
+      registrationPage.emailInput,
+      "email input field should be enabled"
+    ).toBeEnabled();
+
+    await expect(
+      registrationPage.passwordInput,
+      "password input field should be visible"
+    ).toBeVisible();
+    await expect(
+      registrationPage.passwordInput,
+      "password input field should be enabled"
+    ).toBeEnabled();
+
+    await expect(
+      registrationPage.confirmPasswordInput,
+      "confirm password input field should be visible"
+    ).toBeVisible();
+    await expect(
+      registrationPage.confirmPasswordInput,
+      "confirm password input field should be enabled"
+    ).toBeEnabled();
+
+    await expect(
+      registrationPage.registrationButton,
+      "registration button should be visible"
+    ).toBeVisible();
+    await expect(
+      registrationPage.registrationButton,
+      "login button text should have text"
+    ).toHaveText(ApplicationTexts.registrationPage.registrationButtonLabel);
+  });
+
+  test("verify registration form using screenshot", async ({ page }) => {
```

Comment:
Samostatný screenshot by neměl být test. Můžeš ho ale klidně dát do testu `verify registration form is displayed correctly`